asmcomp/wasm32/README.md

```diff
@@ -8,5 +8,52 @@ cmm -> typed cmm -> wasm ast -> linking -> binary
 Typed CMM
 ---
 
+In normal OCaml compilation, after typechecking has been completed the types are removed,
+and thus the cmm representation does not include types.
+However, for WASM, adding some types back at this stage is helpful to cleanly generate the WebAssembly,
+because CMM is closer to WASM than it is to the typed AST.
+WebAssembly types and helper functions are defined in [typed_cmm.ml](typed_cmm.ml).
+
 Linking
 ---
+
+We currently depend on a fork of LLVM for linking with other WASM code.
```

Comment:
The fork is only required for wasm gc, we want to upstream the work at some point.